README.md

```diff
@@ -52,8 +52,8 @@ for the [Wikidata project](https://wikidata.org/).
 ## Release notes
 
 ### 1.1.0 (dev)
-* Renamed `DataTypes_VERSION` constant to `DATATYPES_VERSION`, leaving a deprecated alias.
 * Updated internationalizations.
+* Updated to use the ESLint and Wikibase code sniffers.
```

Comment:
Not really, but the release notes would look quite empty otherwise. I'm fine with removing this line if you think it's better to not have it here.